text/0016-named-tracers.md

```diff
@@ -66,22 +76,42 @@ Examples (based on existing contribution libraries from OpenTracing and OpenCens
 
 ## Internal details
 
-By providing a `TracerFactory` / `MeterFactory` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
+By providing a `TracerProvider` / `MeterProvider` and *Named Tracers / Meters*, a vendor or OpenTelemetry implementation gains more flexibility in providing Tracers and Meters and which attributes they set in the resulting Spans and Metrics that are produced.
 
 On an SDK level, the SpanData class and its Metrics counterpart are extended with a `getLibraryResource` function that returns the resource associated with the Tracer / Meter that created it.
 
+## Glossary of Terms
+
+#### Instrumentation library
```

Comment:
I have heard the opinion that "instrumentation library" is yet another different thing, distinct from "instrumenting library" but synonymous to "telemetry library": https://github.com/open-telemetry/opentelemetry-proto/pull/94#issuecomment-574372313. I would love for instrumentation library to mean the same as instrumenting library. Still I think we might want to consistently use "instrument**ing**" and avoid "instrument**ation**".

Reply:
Instrumenting is simply the present tense verb form of instrument. When you are instrumenting something you are performing the process of instrumentation. If you want to get technical about the difference between instrumentation and telemetry, instrumentation is the process of attaching instruments and telemetry is the process of recording data from the instrumentation. In this sense, the instrumentation library definition here is correct.

That being said, I think it is a mistake to get this pedantic about definitions in the spec as it will just be confusing to people.

Reply:
I completely agree with you @dyladan. 😃 There just seems to be some conflation of "telemetry library" (i.e. the OpenTelemetry library itself) and "instrumenation library" right now, see also https://github.com/open-telemetry/opentelemetry-proto/pull/94#discussion_r367148777.

That being said, I'm also fine with leaving the OTEP as it is now.